specification/dimensions/servicename.md

```diff
@@ -1,7 +1,8 @@
 # Service Name
 
-Service Name is an identifier assigned to a service by the provider. The Service Name is commonly used for cost
-reporting, analysis and allocation scenarios.
+A service represents the offering that was purchased through the provider (e.g., cloud or professional services). For cloud resources, the service is a higher-level classification of the resource type and may include multiple usage charges of different types (e.g., compute, storage, networking). The service is not a classification of the type of usage. Instead, it represents the offering you might find on a provider's website.
+
+The Service Name is a friendly name for the service or offering that was purchased. The Service Name is commonly used for scenarios like analyzing aggregate cost trends over time and filtering data to investigate anomalies.
```

Comment:
Should we use 'display name'? since thats what we've used for other name columns

Reply:
I thought about that. I didn't because there's no corresponding ID. That said, I'm fine with that for consistency.